npm/css-src/sass/_drag-and-drop.scss

```diff
@@ -11,7 +11,7 @@
   transition: outline-offset 0.1s ease-in-out, background-color 0.1s linear;
 }
 
-.drag-and-drop__dropzone [type=file] {
+.drag-and-drop__dropzone [type='file'] {
   display: none;
   left: 10.1em;
   position: relative;
```

Comment:
Yes, we need to keep it. After digging into this code and understanding how it's implemented, it's actually done in a tricky but clever way to give the illusion of drag and drop (as have other people I've seen when I checked online). I'd need to have a talk with you about this because I've tried a few things and have come to the conclusion that we are going to actually have to display it (but make it non-visible for the user), in order for the drag and drop to work.